examples/client/Locomotion/src/pages/ActiveRide/RideDrawer/RideOptions/ServiceOptions/ServiceCard/index.js

```diff
@@ -100,12 +103,18 @@ const ServiceCard = ({ service, withBorder }) => {
       return i18n.t('rideDetails.unavailable');
     }
 
+    // console.log(" Service: " , service); // checkBA? Selected Account? If I have businessAcountId its bussine from newRIde?
     return showPrice ? getFormattedPrice(service.currency, service.price) : null;
   };
 
   useEffect(() => {
-    loadShowPrice();
-  }, []);
+    if (businessAccountId) {
+      const { showPriceToMembers } = getBusinessAccountById(businessAccountId);
+      loadShowPrice(showPriceToMembers ?? false);
+    } else {
+      loadShowPrice();
+    }
```

Comment:
lets extract it to a util resusable funcion that gets  as params
{ loadShowPrice, getBusinessAccountById, businessAccountId } to reduce code duplication